src/ui/public/kuery/suggestions/conjunction.js

```diff
@@ -0,0 +1,22 @@
+const type = 'conjunction';
+
+const conjunctions = {
+  and: `<p>Requires that <span class="suggestionItem__callout">both arguments</span> joined together both return true</p>`,
```

Comment:
This reads a little strangely to me what about just `Requires both arguments to be true`?